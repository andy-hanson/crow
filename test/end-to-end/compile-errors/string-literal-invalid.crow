main void()
	info log "a\0b"
	info log "{"a\0b"::symbol}"
