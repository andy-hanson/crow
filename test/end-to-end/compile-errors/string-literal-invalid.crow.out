exit code: 0
stdout:
info: a
info: a
stderr:
test/end-to-end/compile-errors/string-literal-invalid.crow 2:14-2:20 'string'  literal can't contain '\0'.
test/end-to-end/compile-errors/string-literal-invalid.crow 3:16-3:22 'symbol'  literal can't contain '\0'.
